fv3_cap.F90

```diff
@@ -716,21 +742,35 @@ subroutine InitializeAdvertise(gcomp, rc)
                 if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
               else
                 ! this is a Store() for the first wrtComp -> must do the Store()
-                call ESMF_TraceRegionEnter("ESMF_FieldBundleRegridStore()", rc=rc)
-                call ESMF_FieldBundleRegridStore(fcstFB(j), wrtFB(j,1), &
-                                                 regridMethod=regridmethod, routehandle=routehandle(j,1), &
-                                                 unmappedaction=ESMF_UNMAPPEDACTION_IGNORE, &
-                                                 srcTermProcessing=isrcTermProcessing, rc=rc)
-
-  !             if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
-                if (rc /= ESMF_SUCCESS) then
-                  write(0,*)'fv3_cap.F90:InitializeAdvertise error in ESMF_FieldBundleRegridStore'
-                  call ESMF_LogWrite('fv3_cap.F90:InitializeAdvertise error in ESMF_FieldBundleRegridStore', ESMF_LOGMSG_ERROR, rc=rc)
-                  call ESMF_Finalize(endflag=ESMF_END_ABORT)
+                if (fieldbundle_is_restart) then
+                  call ESMF_TraceRegionEnter("ESMF_FieldBundleRedistStore()", rc=rc)
+                  call ESMF_FieldBundleRedistStore(fcstFB(j), wrtFB(j,1), &
+                                                   routehandle=routehandle(j,1), &
+                                                   rc=rc)
+                  if (rc /= ESMF_SUCCESS) then
+                    write(0,*)'fv3_cap.F90:InitializeAdvertise error in ESMF_FieldBundleRedistStore at line ',__LINE__
```

Comment:
Do we need this line? The error message should be in PET log files, see line 752.